### PR TITLE
Scheduled livestreams improvements

### LINKED ISSUE #34: Scheduled livestreams improvements
### Issue type

Feature request

For sport events: filter with live or on demand content:
http://www.srfcdn.ch/developer-docs/integrationlayer/api/public/swagger/index.html#/operations/MediaList/MediaListVideoScheduledLivestreamsLivecenter
- `types`
- `onlyEventsWithResult`

For web events: filter video with sign language:
http://www.srfcdn.ch/developer-docs/integrationlayer/api/public/swagger/index.html#/operations/MediaScheduledLivestreamList/MediaListVideoScheduledLivestreams
- `signLanguageOnly`

### Description of the problem

Play applications has to filter it to display in the "livestreams" page.

### PR DESCRIPTION
**Pull request by pyby**
## Description

Resolves #34.

## Changes made

In Objective C, the API has a breaking change, due to additional parameters.

## Checklist

- [x] Your branch has been rebased onto the `develop` branch.
- [x] APIs have been properly documented (if relevant).
- [x] The documentation has been updated (if relevant).
- [x] New unit tests have been written (if relevant).

### CODE REVIEW

File: Sources/SRGDataProviderCombine/LiveCenterServices.swift [hunk 1]
```diff
@@ -15,9 +15,12 @@ import Combine
 public extension SRGDataProvider {
     /**
      *  List of videos available from the Live Center.
+     *
+     *  - Parameter contentTypeFilter: The content type filter to apply.
+     *  - Parameter eventsWithResultOnly: Whether only medias which are in the Live Center results center must be returned. Related to sport events with a cesim id.
```
Comment (defagos) [inline Sources/SRGDataProviderCombine/LiveCenterServices.swift:20]: _Related to sport events with a cesim id_ is probably too technical (I personally don't fully understand what this means).

File: Sources/SRGDataProviderNetwork/include/SRGDataProvider+LiveCenterServices.h [hunk 2]
```diff
@@ -17,8 +17,13 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  *  List of videos available from the Live Center.
+ *
+ *  @param contentTypeFilter    The content type filter to apply.
+ *  @param eventsWithResultOnly Whether only medias which are in the Live Center results center must be returned. Related to sport events with a cesim id.
```
Comment (defagos) [inline Sources/SRGDataProviderNetwork/include/SRGDataProvider+LiveCenterServices.h:22]: Same comment as above

**Review by defagos — CHANGES_REQUESTED** [review]
I reviewed the changes, all good to me (except a minor documentation improvement I suggest). Thanks!

I have no other proposal to make for parameter names either (the underlying webservice API does not make this task any easier).

File: Sources/SRGDataProviderCombine/LiveCenterServices.swift (hunk 1 above)
Comment (pyby) [reply Sources/SRGDataProviderCombine/LiveCenterServices.swift]: Thank you for the feedback. Cesim id is related to all SRGSSR sport api to trigger sport events.
As it's not exposed in the Integration Layer API, I removed the mention.

File: Sources/SRGDataProviderNetwork/include/SRGDataProvider+LiveCenterServices.h (hunk 2 above)
Comment (pyby) [reply Sources/SRGDataProviderNetwork/include/SRGDataProvider+LiveCenterServices.h]: Same comment as above

**Pull request merged**